src/js/popup.js

```diff
@@ -51,11 +51,39 @@ function showSignUpPanel() {
 }
 
 
-function choosePanel(numRemaining, panelId, premium, premiumEnabledString){
-  if (premium && premiumEnabledString === "True"){
+function premiumFeaturesAvailable(premiumEnabledString) {
+  if (premiumEnabledString === "True") {
+    return Boolean(true);
+  }
+  else {
+    return Boolean(false);
+  }
+}
+
+function enablePremium(premium, premiumEnabledString) {
+  if (premium && premiumFeaturesAvailable(premiumEnabledString)) {
+    return Boolean(true);
+  }
+  else {
+    return Boolean(false);
+  }
+}
+
+
+function choosePanel(numRemaining, panelId, premium, premiumEnabledString, premiumSubdomainSet){
+  const premiumPanelWrapper = document.querySelector(".premium-wrapper");
+
+  if (enablePremium(premium, premiumEnabledString)){
```

Comment:
Grouped up all of the features that meet criteria 1) and 2) into an if-statement for tidiness.

1 ) user is premium 2) premium features